From cc1e46f64dcb3e031a56273ceb86a1a4ad2e5d7b Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 6 Jan 2011 16:52:57 +0800
Subject: [PATCH 135/280] ENGR00137445-1 Update Android.mk to add
 LOCAL_MODULE_TAGS := eng

Update Android.mk to add LOCAL_MODULE_TAGS := eng

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 ipu/Android.mk | 2 +-
 vpu/Android.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ipu/Android.mk b/ipu/Android.mk
index 2be3d20..70c3644 100644
--- a/ipu/Android.mk
+++ b/ipu/Android.mk
@@ -14,6 +14,6 @@ LOCAL_SHARED_LIBRARIES := libutils libc
 LOCAL_MODULE := libipu
 LOCAL_LD_FLAGS += -nostartfiles
 LOCAL_PRELINK_MODULE := false
-LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_TAGS := eng
 include $(BUILD_SHARED_LIBRARY)
 
diff --git a/vpu/Android.mk b/vpu/Android.mk
index abd7e6e..177b09e 100755
--- a/vpu/Android.mk
+++ b/vpu/Android.mk
@@ -13,7 +13,7 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)
 LOCAL_MODULE := libvpu
 LOCAL_LD_FLAGS += -nostartfiles
 LOCAL_PRELINK_MODULE := false
-LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_TAGS := eng
 include $(BUILD_SHARED_LIBRARY)
 
 endif
-- 
1.8.0

